### PR TITLE
Fixed test_prayers_integration test due to a difference in the timezone context

### PR DESCRIPTION
**Pull request by albertisfu**
This is a fix for the failing `test_prayers_integration` test, which starts failing from 5 PM PT, corresponding to the next day in UTC. The issue is caused by a difference in the timezone context when formatting the date in the test.

### CODE REVIEW

File: cl/favorites/tests.py [hunk 1]
```diff
@@ -924,10 +924,13 @@ async def test_prayers_integration(self) -> None:
             f"https://www.courtlistener.com{rd_6.get_absolute_url()}",
             email_text_content,
         )
-        self.assertIn(
-            f"You requested it on {template_date(prayer_1.date_created, 'M j, Y')}",
-            email_text_content,
-        )
+        with timezone.override("America/Los_Angeles"):
```
Comment (ttys0dev) [inline cl/favorites/tests.py:927]: Will this break if the system's timezone is different?

**Comment by mlissner** [comment]
I'm happy if you and @ttys0dev are happy. :)

**Comment by mlissner** [comment]
Thanks for catching and fixing.

File: cl/favorites/tests.py (hunk 1 above)
Comment (albertisfu) [reply cl/favorites/tests.py]: good observation. I've changed it to use the timezone user in settings. Thanks

**Comment by albertisfu** [comment]
tweaked it to use the timezone from settings, which is better as @ttys0dev pointed out

**Pull request merged**